### PR TITLE
Add role to install ovirt-guest-agent and ensure service is running

### PR DESCRIPTION
**Pull request by KKoukiou**
One small comment: I added in the ansible-lint check the -x ANSIBLE0010 because I got this error

### CODE REVIEW

File: tox-ansible-lint-test.sh [hunk 1]
```diff
@@ -18,4 +18,4 @@ for path_role in ${TOXINIDIR}/roles/*; do
 done
 
 # Test ansible-lint for every created playbook
-ansible-lint ${TOXWORKDIR}/playbooks/*.yml
+ansible-lint -x ANSIBLE0010 ${TOXWORKDIR}/playbooks/*.yml
```
Comment (lukas-bednar) [inline tox-ansible-lint-test.sh]: I would rather put skip-list tag to relevant tasks instead of disable some warning for all roles.

Comment (KKoukiou) [inline tox-ansible-lint-test.sh]: Generally I agree but this warning does not make sense to me, especially when the documentation suggests using yum with state==latest.
But I can change it.

Comment (lukas-bednar) [inline tox-ansible-lint-test.sh]: After the change you can try to raise this issue to ansible-lint project ;-)

**Comment by KKoukiou** [comment]
@StLuke Regarding the comments about the rhevm-guest-agent-common package that should not have the common suffix I think it should. It's documented like this here [https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Virtualization/3.5/html/User_Guide/Installing_the_Guest_Agents_and_Drivers_on_Red_Hat_Enterprise_Linux.html](url) but also when I yum search for the package without the common suffix I get 0 results. Please confirm and I 'll revert it to use -common suffix.

**Comment by StLuke** [comment]
1. thats 3.5 docuemntation you should use `yum install ovirt-guest-agent` which will install ovirt-guest-agent-common
2. service is called ovirt-guest-agent not ovirt-guest-agent common in 4.1
2. rhevm-guest-agent does not exist anymore (was moved to upstream ovirt version)
3. if rhevm-guest-agent would exist we still wants to use ovirt-guest-agent as these are playbooks for oVirt not for RHEVM, if you want to use it with previous version of dowstream guest agent (rhevm-guest-agent) parametrize it and add it in your inventory vars

**Comment by KKoukiou** [comment]
@StLuke Thanks for the comments, I parameterized the prefix as you suggested. Let me know if any other change is needed.

File: roles/ovirt-guest-agent/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Requirements
+------------
+
+Configured rhev repository.
+
+
+Role Variables
+--------------
+
+---
```
Comment (lukas-bednar) [inline roles/ovirt-guest-agent/README.md]: please make it nice highlighted as yaml snippet. like:

```yaml
---
guest_agent_pkg_prefix: "ovirt|rhevm|qemu"
```

File: roles/ovirt-guest-agent/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Target Systems
+--------------
+
+* rhv-guest
+
+
+Requirements
+------------
+
+Configured rhev repository.
```
Comment (lukas-bednar) [inline roles/ovirt-guest-agent/README.md]: ... ovirt repository

File: roles/ovirt-guest-agent/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+Ovirt guest-agent install package
```
Comment (lukas-bednar) [inline roles/ovirt-guest-agent/README.md]: oVirt

File: roles/ovirt-guest-agent/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+=============================
+
+This role installs {ovirt|rhevm|qemu}-guest-agent package and
+ensures the service is running.
+
+Target Systems
+--------------
+
+* rhv-guest
```
Comment (lukas-bednar) [inline roles/ovirt-guest-agent/README.md]: ovirt-guest

File: roles/ovirt-guest-agent/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+Configured rhev repository.
+
+
+Role Variables
+--------------
+
+---
+guest_agent_pkg_prefix: "ovirt|rhevm|qemu"
+
+Default value if variable not given is ovirt.
```
Comment (lukas-bednar) [inline roles/ovirt-guest-agent/README.md]: Default value if variable not given is `ovirt`.

File: roles/ovirt-guest-agent/README.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+------------
+
+None
+
+Example Playbook
+----------------
+
+```yaml
+---
+- hosts: rhv-guest
```
Comment (lukas-bednar) [inline roles/ovirt-guest-agent/README.md]: ovirt-guest

File: roles/ovirt-guest-agent/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+Configured ovirt repository.
+
+
+Role Variables
+--------------
+
+```yaml
+---
+guest_agent_pkg_prefix: "ovirt|rhevm|qemu"
```
Comment (StLuke) [inline roles/ovirt-guest-agent/README.md]: s/guest-agent_pkg_prefix/ovirt-guest-agent_pkg_prefix/

as its role_name_var_name

File: roles/ovirt-guest-agent/handlers/main.yaml [hunk 9]
```diff
@@ -0,0 +1,6 @@
+---
+- name: enable and start {{ guest_agent_pkg_prefix }}-guest-agent
```
Comment (StLuke) [inline roles/ovirt-guest-agent/handlers/main.yaml]: s/guest-agent_pkg_prefix/ovirt-guest-agent_pkg_prefix/

as its role_name_var_name

File: roles/ovirt-guest-agent/handlers/main.yaml [hunk 10]
```diff
@@ -0,0 +1,6 @@
...
+  service:
+    name: "{{ guest_agent_pkg_prefix }}-guest-agent"
```
Comment (StLuke) [inline roles/ovirt-guest-agent/handlers/main.yaml]: s/guest-agent_pkg_prefix/ovirt-guest-agent_pkg_prefix/

as its role_name_var_name

File: roles/ovirt-guest-agent/meta/main.yml [hunk 11]
```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: "Katerina Koukiou"
+  description: "Installs guest agents on RHEL"
```
Comment (StLuke) [inline roles/ovirt-guest-agent/meta/main.yml]: s/agents/agent/

on RHEL? officially supported for oVirt is CentOS and Fedora I believe so I would just s/on RHEL/on VM/

File: roles/ovirt-guest-agent/tasks/main.yml [hunk 12]
```diff
@@ -0,0 +1,9 @@
+- name: Install latest {{ guest_agent_pkg_prefix }}-guest-agent
```
Comment (StLuke) [inline roles/ovirt-guest-agent/tasks/main.yml]: s/guest-agent_pkg_prefix/ovirt-guest-agent_pkg_prefix/

as its role_name_var_name

File: roles/ovirt-guest-agent/tasks/main.yml [hunk 13]
```diff
@@ -0,0 +1,9 @@
...
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - "{{ guest_agent_pkg_prefix }}-guest-agent"
```
Comment (StLuke) [inline roles/ovirt-guest-agent/tasks/main.yml]: s/guest-agent_pkg_prefix/ovirt-guest-agent_pkg_prefix/

as its role_name_var_name

File: roles/ovirt-guest-agent/tasks/main.yml [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+  notify: enable and start {{ guest_agent_pkg_prefix }}-guest-agent
```
Comment (StLuke) [inline roles/ovirt-guest-agent/tasks/main.yml]: s/guest-agent_pkg_prefix/ovirt-guest-agent_pkg_prefix/

as its role_name_var_name

File: roles/ovirt-guest-agent/README.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
+oVirt guest-agent install package
+=============================
+
+This role installs {ovirt|rhevm|qemu}-guest-agent package and
```
Comment (StLuke) [inline roles/ovirt-guest-agent/README.md]: s/{ovirt|rhevm|qemu}-guest-agent/ovirt-guest-agent/

By default it install ovirt-guest-agent, parametrizing is mentioned later

File: roles/ovirt-guest-agent/README.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+ensures the service is running.
```
Comment (StLuke) [inline roles/ovirt-guest-agent/README.md]: enabled and running

**Comment by lukas-bednar** [comment]
@KKoukiou Please add link to main README to include your role there.

**Pull request merged**